### ISSUE TITLE
flatbuffers-dev missing

### ISSUE THREAD

**Issue by erhan-** [satchm0h/alpine-snort3#1]
```
 ---> Running in 88f13e0322b6                                                                                                                                                                 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz                                                                                                                   
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz                                                                                                              
fetch http://nl.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz                                                                                                                    
v3.8.5-66-gccbd6a8ae7 [http://dl-cdn.alpinelinux.org/alpine/v3.8/main]                                                                                                                        
v3.8.5-66-gccbd6a8ae7 [http://dl-cdn.alpinelinux.org/alpine/v3.8/community]
v20210212-440-gd6b8774b17 [http://nl.alpinelinux.org/alpine/edge/testing]
OK: 13863 distinct packages available
OK: 4 MiB in 13 packages                                                                                                                                                                      
Removing intermediate container 88f13e0322b6                                                                                                                                                  
 ---> 819b1c1143d1
Step 6/36 : RUN apk add --no-cache     wget     build-base     git     cmake     bison     flex     lcov@testing     cppcheck     cpputest     flatbuffers-dev@testing     libdnet-dev     libpcap-dev     libtirpc-dev     luajit-dev     libressl-dev     zlib-dev     pcre-dev     libuuid     xz-dev
 ---> Running in 1644c69fcbde                                                                                                                                                                 
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/main/x86_64/APKINDEX.tar.gz                                                                                                                   
fetch http://dl-cdn.alpinelinux.org/alpine/v3.8/community/x86_64/APKINDEX.tar.gz                                                                                                              
fetch http://nl.alpinelinux.org/alpine/edge/testing/x86_64/APKINDEX.tar.gz                                                                                                                    
ERROR: unsatisfiable constraints:                                                                                                                                                             
  flatbuffers-dev (missing):                                                                                                                                                                  
    required by: world[flatbuffers-dev]
ERROR: Service 'snort' failed to build: The command '/bin/sh -c apk add --no-cache     wget     build-base     git     cmake     bison     flex     lcov@testing     cppcheck     cpputest     flatbuffers-dev@testing     libdnet-dev     libpcap-dev     libtirpc-dev     luajit-dev     libressl-dev     zlib-dev     pcre-dev     libuuid     xz-dev' returned a non-zero code: 1
```

It is available in alpine:3.13 but there are still some errors during compiling. 
cpputest libs are removed for example in the latest package but needed for compiling snort3:
https://gitlab.alpinelinux.org/alpine/aports/-/issues/11908

**Comment by satchm0h** [comment]
For the time being, I fixed the build in Alpine 3.8. The flatbuffers dependency is superfluous. While I was in there I pointed the build to the new DAQ library. 

If I can find the time, I will attempt to update the Dockerfile to work on a newer Alpine release. 

Thanks for the issue.

**Comment by erhan-** [comment]
No, thank you! I will try it again after work!

Edit: it works! Thank you!
